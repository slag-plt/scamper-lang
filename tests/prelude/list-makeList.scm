(list "ab" "ba")
(list 1 2 3)
(list)
(list #t #t #t #t)
(list #f #f #f #f)
(make-list 2 "a")
(make-list 10 "HelloWorld")
(make-list 5 10)
(make-list 2 #t)
(make-list 2 "abab")
(make-list 0 "")