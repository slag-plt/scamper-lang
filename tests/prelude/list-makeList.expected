(list "ab" "ba")
(list 1 2 3)
null
(list #t #t #t #t)
(list #f #f #f #f)
(list "a" "a")
(list "HelloWorld" "HelloWorld" "HelloWorld" "HelloWorld" "HelloWorld" "HelloWorld" "HelloWorld" "HelloWorld" "HelloWorld" "HelloWorld")
(list 10 10 10 10 10)
(list #t #t)
(list "abab" "abab")
null
